ceph/rbd_mirror_status.go

```diff
@@ -0,0 +1,176 @@
+//   Copyright 2022 DigitalOcean
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package ceph
+
+import (
+	"encoding/json"
+	"os/exec"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/sirupsen/logrus"
+)
+
+const rbdPath = "/usr/bin/rbd"
+
+const (
+	// RbdMirrorOK denotes the status of the rbd-mirror when healthy.
+	RbdMirrorOK = "OK"
```

Comment:
Just a general question, do these strings change between Nautilus and Pacific?  Same question about the struct layout below.

Reply:
the struct actually doesn't have daemon and image health separate, so I need to double check/test that on nautilus.

Reply:
yes, still works fine on nautilus